pkg/server/systems.go

```diff
@@ -0,0 +1,72 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/google/uuid"
+)
+
+type system struct {
+	ID          uuid.UUID
+	Acronym     string
+	Name        string
+	Description string
+	Datapoint1  string
+	Datapoint2  string
+	Datapoint3  string
+	Datapoint4  string
+}
+
+type systems []system
+
+func (s *server) handleSystemsList() http.HandlerFunc {
```

Comment:
Somehow I wasn't able to trigger the `/systems` endpoint to return anything. I got `http: panic serving [::1]:61204: runtime error: index out of range [1] with length 1`. I'm wondering how you tested this. Also, I think this file could use a test, similar to the landing test.

Reply:
That's weird, I hooked it up and it worked okay for me. I'll add a test.